app/Http/Controllers/FaqController.php

```diff
@@ -41,8 +42,8 @@ public function update($id, Request $request)
         ]);
 
         $faq                 = $this->faq->where('id', $id);
-        $faq->question       = $request->question;
-        $faq->answer         = $request->answer;
+        $faq->question       = $request->question->where('id', $id);
```

Comment:
the previous implementation is correct, no need for the where n question and answer

Reply:
I fixed it.